src/main/java/seedu/address/logic/Anakin_commands/AnakinListCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.Anakin_commands;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Anakin_Model;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class AnakinListCommand extends Anakin_Command {
```

Comment:
Should not be in this pull req

Reply:
Oh yeah this change got squashed into this PR accidentally.